mappings/servers/dkgames/metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+    "id": "dkgames",
+    "name": "DKGames",
+    "description": "Serveur DKGames",

```

Comment:
Does your server have an MOTD different to this that represents your server? If so, I recommend putting that as your description.

Reply:
this is my motd https://fr.namemc.com/server/play.dkgames.fun what du u recommend to put in description ?